homeassistant/components/dialogflow/__init__.py

```diff
@@ -84,23 +87,45 @@ class DialogFlowError(HomeAssistantError):
 
 def dialogflow_error_response(message, error):
     """Return a response saying the error message."""
-    dialogflow_response = DialogflowResponse(message["result"]["parameters"])
+    _api_version = get_api_version(message)
```

Comment:
Please don't prefix parameters with `_` if you're using them.

Reply:
Agreed - that was bad styling on my part. It's fixed now.